docs/submit_to_alibaba_cloud_realtime_compute.md

```diff
@@ -0,0 +1,42 @@
+# Submit Feathub Job to Alibaba Cloud Realtime Compute for Apache Flink(实时计算Flink版)
```

Comment:
How about moving this README to `docs/how_to_guides/alibaba_cloud_deployment.md`?